src/web/nextui/src/app/eval/ResultsTable.tsx

```diff
@@ -316,8 +295,31 @@ function EvalOutputCell({
     } catch (error) {
       console.error('Invalid regular expression:', (error as Error).message);
     }
-  } else if (renderMarkdown && !hasImageLightbox) {
-    node = <ReactMarkdown>{text}</ReactMarkdown>;
+  } else if (renderMarkdown) {
+    if (!output.pass && text.includes('---')) {
+      chunks = text.split('---');
+      text = chunks.slice(1).join('---');
+    } else {
+      chunks = [text];
+    }
```

Comment:
Oh wait, we probably want this regardless of markdown rendering. Will hoist it out of this conditional